Packs/Gamma/Integrations/Gamma/README.md

```diff
@@ -0,0 +1,118 @@
+Gamma Enterprise DLP provides 1-click automatic discovery and remediation of data loss instances
+ across enterprise sanctioned SaaS applications (cloud and on-prem) such as: Slack, Github, GSuite (Gmail, GDrive), Atlassian Suite (Jira, Confluence), Microsoft Office 365 (Outlook, Teams, OneDrive), ServiceNow, ZenDesk and many more.
+
+## Configure Gamma.AI Enterprise DLP on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Gamma.
+3. Click **Add instance** to create and configure a new integration instance.
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| api_key | Gamma Discovery API Key | True |
+
+Click **Test** to validate the URLs, token, and connection.
+
+## Commands
+You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
+After you successfully execute a command, a DBot message appears in the War Room with the command details.
+
+### fetch-incidents
+***
+Fetch DLP violations found across SaaS applications monitored by Gamma 
+
+#### Base Command
+
+`fetch-incidents`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| first_fetch_violation | Violation id offset. | Optional | 
+| max_results | Default is "10". | Optional | 
+
+
+#### Context Output
+
+| **Field** | **Type** | **Description** |
+| --- | --- | --- |
+| response.violation_id | Integer | Violation ID | 
+| response.file_labels_map | Array | File in reference to the DLP violation | 
+| response.violation_status | String | one of 'OPEN', 'RESOLVED', 'IGNORED' | 
+| response.violation_category | String | Category of the violation e.g. PII, Secrets, GDPR/CCPA, etc. | 
+| response.violation_event_timestamp | Integer | Timestamp of violation in epoch milliseconds | 
+| response.text_labels | Array | Data classification labels |
+| response.app_name | String | Name of the application |
+| response.dashboard_url | Array | Gamma dashboard URL |
+| response.user | JSON Object | a JSON field containing optional information (based on what the app allows us to access) like email address, name, atlassian account id, AD id, github login, etc. All these fields are nullable. |
+
+#### Command Example
+```!fetch-incidents first_fetch_violation=998 max_results=1```
+
```

Comment:
I think that you've meant to do that for the `gamma-get-violation-list` command and not the `fetch-incidents` since it's not a command of the integration.

Reply:
Yep, fixed it